postcode_lookup/template_sorter.py

```diff
@@ -141,21 +189,15 @@ def weight(self):
             return -6000
 
         if self.timetable.is_after(TimetableEvent.REGISTRATION_DEADLINE):
-            return 1001
+            return 1002
         return 0
 
-    @property
+    @cached_property
```

Comment:
I switched all the `context` properties to cached properties in 8e010e930f4ab364190a09a18a72c4d6d6321f21

In a template like the one I've added where we call `section.context.someproperty` lots and lots of times, we call these methods over and over. None of them are really computationally intensive, but it seems wasteful given they return the same data every time.

Reply:
Just to double check: this is cached at the class instance level, right? So we don't need to worry about the same Lambda instance serving more than one request / threading issues?

Reply:
Yep. It is cached on the object/instance and lasts for the lifetime of the instance. New object, clear cache